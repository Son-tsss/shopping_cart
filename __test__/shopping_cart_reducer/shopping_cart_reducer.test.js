import shoppingCartReducer from "../../src/shopping_cart/shopping_cart_reducer"

it('new shopping cart created with empty items list', () => {
    const real = shoppingCartReducer();

    expect(real).toMatchSnapshot();
});
