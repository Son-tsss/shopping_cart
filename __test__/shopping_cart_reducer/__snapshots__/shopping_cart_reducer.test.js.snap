// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`new shopping cart created with empty items list 1`] = `
Object {
  "discount": 1,
  "items": Array [],
}
`;
